pyaerocom/aeroval/fairmode_stats.py

```diff
@@ -77,7 +77,7 @@ def fairmode_stats(obs_var: str, stats: dict) -> dict:
     assert np.isclose(
         rmsu * beta_mqi,
         np.sqrt((bias) ** 2 + (mod_std - obs_std) ** 2 + (2 * obs_std * mod_std * (1 - R))),
-        rtol=1e-3,
+        rtol=1e-2,
```

Comment:
This is related to the rounding done for the statistical values.
https://github.com/metno/pyaerocom/commit/dc40c69ae9a73f0a413ae1ae2f76c58a3819376c reduced the accuracy from 1e-10 to 1e-6. Basically, the formula written there is not true for rounded values. I don't know why it is importand for fairmode. 

I am not sure what the test here tries to achieve? It compares two different calculations for rms (beta_mqi = rms/rmsu), but the second one with rounded values?